### PR TITLE
liferea: update to 1.14.5.

### PR DESCRIPTION
**Pull request by geekdiop**
Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by duskull88** [comment]
There is 1.14.4 now, but it cannot be compiled on Void, as discussed here: `https://github.com/lwindolf/liferea/issues/1209`

**Comment by Chocimier** [comment]
1.14.4 builds within xbps-src just fine

File: srcpkgs/liferea/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'liferea'
 pkgname=liferea
-version=1.12.9
+version=1.14.0
```
Comment (zen0bit) [inline srcpkgs/liferea/template:3]: version=1.14.5

File: srcpkgs/liferea/template [hunk 2]
```diff
@@ -15,7 +15,7 @@ license="GPL-2.0-or-later"
 homepage="https://lzone.de/liferea/"
 changelog="https://raw.githubusercontent.com/lwindolf/liferea/master/ChangeLog"
 distfiles="https://github.com/lwindolf/liferea/releases/download/v${version}/${pkgname}-${version}.tar.bz2"
-checksum=d8180695f0d378ad2799b36230bc8662916667d8a09b2e7144393a2a79c8cb1b
+checksum=b82de4b09e27357063418a8d392e2a2ba6822bafd6cdfdbb6171db33bdd3a5d5
```
Comment (zen0bit) [inline srcpkgs/liferea/template:18]: checksum=2244e78d8f6b6e28f411a3554d61496a012d5fe1392fe1b475978ecfd8c61177

**Review by zen0bit — CHANGES_REQUESTED** [review]
new stable version out...

File: srcpkgs/liferea/template (hunk 2 above)
Comment (zen0bit) [inline srcpkgs/liferea/template:18]:
```suggestion
checksum=2244e78d8f6b6e28f411a3554d61496a012d5fe1392fe1b475978ecfd8c61177
```

**Review by zen0bit — CHANGES_REQUESTED** [review]
1.14.5 stable is out

**Comment by Duncaen** [comment]
Merged as https://github.com/void-linux/void-packages/commit/00c969b44c4963c54e8a08c8ef982ed64bdbe58c.

**Pull request closed without merging**